app/connector/jira/pom.xml

```diff
@@ -32,11 +32,18 @@
         <!-- The 5.1.0 version uses an old guava 20 version incompatible with the default guava set on syndesis -->
         <atlassian-jira-rest-java-client-version>5.1.6</atlassian-jira-rest-java-client-version>
         <atlassian-util-concurrent-version>4.0.1</atlassian-util-concurrent-version>
+
+        <!-- TODO: Remove when using Camel 3 bits in Syndesis -->
```

Comment:
Discussed in live conversation, the Jira component is here while waiting the backport to 2.23 camel branch, we can live with this in the while.

Reply:
yes @claudio4j is already working on that. Once he is finished with backporting the camel-jira component to 2.23.x we can remove the component clone again from Syndesis.

Reply:
This is the PR we need to align 2.21 and 2.23: https://github.com/jboss-fuse/camel/pull/1185